cdap-app-fabric/src/test/java/io/cdap/cdap/metadata/FakeFieldLineageReader.java

```diff
@@ -62,6 +62,11 @@
     return fields;
   }
 
+  @Override
+  public Set<String> getDroppedFields(EndPoint endPoint, long start, long end) {
+    return null;
```

Comment:
Does the interface contract allows `null`? Also, generally it's better to use empty collection instead of null.